### DISCUSSION TITLE
Nuclei to use for scan orchestration of various tools

### DISCUSSION THREAD

**Discussion by jakre1234** [projectdiscovery/nuclei#1768] [Ideas]
Hey, 
The way I am looking at nuclei evolving day by day, can we expect it to be complete all in one solution to be the first priority security tool.
I see the primary usage of nuclei is template based web scanner but I was wondering it these capiblities be extneded to write custom exploit or econ modules and adding them in a workflow (something like osmedus). Yes osmedus( https://github.com/osmedeus/osmedeus-base , I cam across this a few weeks back), it seems promising but a bit sophisticated while writing workflows and modules.
These can be used like users can write different modules from subdomain enumeration to web screenshot to waybackurls and adding them in a workflow such that it executes parallel y(wherever possible), something like,
flow : reconMaster
  -module:
      -sub_enum    // runs in parallel
     - waybackurls
  -module:
      -screenshot

**Comment by ehsandeep** [comment]
@jakre1234 thank you for sharing this. Nuclei cannot be used for scan orchestration of various tools at this time, however, it is a good idea to discuss the possibility of doing so in the future. I've moved this issue to the discussion/idea section to keep the conversation going.

**Comment by jakre1234** [comment]
Thanks for the information.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
